src/command.js

```diff
@@ -48,9 +48,12 @@ export default class StorybookCommand extends Command {
     });
 
     let url = await this.storybook();
-    let pages = await this.getStoryPages(url);
+    // borrow a browser page to discover stories
```

Comment:
Can probably update this comment

```suggestion
    // borrow a browser page to get the storybook version and discover stories
```